docker/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '2.1'
+
+services:
+  pvw:
+    image: 135080795405.dkr.ecr.us-east-1.amazonaws.com/enlil:pvw-enlil-osmesa
```

Comment:
Did you want to use our public URL here: 
public.ecr.aws/enlil/paraview_web_repo:pvw-enlil-osmesa?

Reply:
Yep, good catch! I was testing earlier and wasn't sure whether that was the issue or not, so kept using the original one.
I just pushed up that change.